server/service/src/vaccine_course/insert.rs

```diff
@@ -37,9 +49,30 @@ pub fn insert_vaccine_course(
         .connection
         .transaction_sync(|connection| {
             validate(&input, connection)?;
-            let new_vaccine_course = generate(input);
+            let new_vaccine_course = generate(input.clone());
             VaccineCourseRowRepository::new(connection).upsert_one(&new_vaccine_course)?;
 
+            // Update ITEMS - Delete and recreate all records.
+            // If nothing has changed, we still need to query and compare each record so this is the simplest way
```

Comment:
I'm bringing across explanatory comments from elsewhere where we are using the same code.

I kind of want to get rid of it, but then maybe it wouldn't make sense?

Reply:
I think the comments are helpful